autogen/oai/cohere.py

```diff
@@ -67,6 +68,7 @@ def __init__(self, **kwargs):
         """
         # Ensure we have the api_key upon instantiation
         self.api_key = kwargs.get("api_key", None)
+        self.client_name = kwargs.get("client_name") or "autogen-cohere"
```

Comment:
Cool, can I suggest:
```suggestion
        self.client_name = kwargs.get("client_name", "autogen-cohere")
```

Reply:
Hmm... In this case, if client_name is None or "" in the configs itself, then it'll be empty...

Reply:
Ideally we don't want that to happen, which is why I used or

Reply:
Ah, okay, let's leave it as you suggested, thanks